### DISCUSSION TITLE
Editable column does not refresh using events

### DISCUSSION THREAD

**Discussion by jwhoami** [Power-Components/livewire-powergrid#514] [Q&A]
Hi,
I have defined 2 columns, username is not editable and the name is editable (editOnClick), like so...

`      Column::make('USERNAME', 'username')
        ->sortable()
        ->searchable()
        ->makeInputText(),
      Column::make('NAME', 'name')
        ->sortable()
        ->searchable()
        ->makeInputText()
        ->editOnClick(true),
`

Using wire elements modal, I edit the username and name and after updating I emit the 'pg:eventRefresh-default' event.
The username is refreshed while the name is not refreshed. I have to refresh the browser to see the updated name.

Is this a bug or am I doing something wrong (which might be the case most probably).

Thanks

**Comment by gallomaf** [comment]
+1
